docs/source/contributing/using_gitpod.md

```diff
@@ -0,0 +1,73 @@
+(using_gitpod)=
+# Using Gitpod
+
+## About Gitpod
+[Gitpod](https://www.gitpod.io/) is a browser-based development environment.
+
+These are some benefits to using Gitpod:
+
+- Bypass local computer configuration and technical issues
+- Save time by using a pre-configured virtual environment for contributing to open source
+- Save space on your local computer
+- Alleviate delays in situations with low internet bandwidth
+
+## Using Gitpod to Contribute to PyMC
+
+### Gitpod Workflow
+1. Fork the pymc repo: https://github.com/pymc-devs/pymc
+
+2. Create a Gitpod account. You can login and authorize access via your GitHub account:  https://gitpod.io/
+
+NOTE: Gitpod will show up as an authorized application in your account here:  https://github.com/settings/applications
+
+3. Grant GitHub / Gitpod integration permissions.
+
+a) Go to: https://gitpod.io/user/integrations
+
+b) Select GitHub and then "edit permissions"
+
+c) Select these permission: user:email, public_repo, repo, workflow
+
+<!-- ![gitpod-integration](gitpod/gitpod_integration.png) -->
+
+![gitpod_integration](gitpod/gitpod_integration.png)
+
```

Comment:
@OriolAbril 
I am thinking this is not the ideal way to include images?
I tried to specify size via Markdown, but that didn't work.